json/Card/seeking-the-truth.json

```diff
@@ -0,0 +1,27 @@
+[
+    {
+        "clan": "neutral",
+        "cost": null,
+        "deck_limit": 1,
+        "element": "water",
```

Comment:
is that a 100% sure guess?

Reply:
Seems to be a cycle, each pack is themed with an element, has a province restricted to a role, province has the same role restriction and element affiliation for Air and Earth at least.
[Air province](https://images-cdn.fantasyflightgames.com/filer_public/75/a6/75a6ef57-7703-48df-9ebe-390c437b99f8/l5c09_cardfan_a1.png)
[Earth province](https://images-cdn.fantasyflightgames.com/filer_public/04/0f/040f1b11-12f0-4d59-87bb-5b2d375c2a8f/l5c10_cardfan_a1.png)